content/docs/studio/user-guide/projects-and-experiments/explore-ml-experiments.md

```diff
@@ -98,39 +106,56 @@ Commits can be hidden from the project table in the following ways:
   commits that do not add much value in your project. To hide a commit or
   branch, click on the 3-dot menu next to the commit or branch name and click on
   `Hide commit` or `Hide branch`.
+
+  ![Hide commit](https://static.iterative.ai/img/studio/hide_commit.png)
+
 - **Unhide commits:** You can unhide commits as needed, so that you don't lose
   any experimentation history. To display all hidden commits, click on the
-  `Show hidden commits` toggle (refer [the above gif](#display-preferences)).
-  This will display all hidden commits, with a `hidden` (closed eye) indicator.
+  `Show hidden commits` toggle (refer [filters](#filters)). This will display
+  all hidden commits, with a `hidden` (closed eye) indicator.
+
+  ![Hidden commit indicator](https://static.iterative.ai/img/studio/hidden_commit_indicator.png)
+
   To unhide any commit, click on the 3-dot menu for that commit and click on
   `Show commit`.
 
-### Selected only:
+  ![Show hidden commit](https://static.iterative.ai/img/studio/show_hidden_commit.png)
 
-Toggle between showing and hiding experiments that you have not selected.
+### Delta mode
 
-### Delta mode:
+You can display either the absolute values of the columns (metrics,
+hyperparameters, etc) or their delta (difference) from the baseline row. To
+toggle between these two options, use the `Delta mode` button.
```

Comment:
```suggestion
toggle between these two options, use the `Delta mode` button:
```
There might be more instances.
Maybe it's a stylistic choice, but it's very weird for me to see a sentence end in a period (`.`) if it is explicitly demonstrated by an image right underneath it (expect a `:`)

Reply:
Not feeling strongly (this should not block this PR to be clear), but it does look weird IMO (going from `.` to an image demonstrating the above description).
I would suggest on aligning on `:`  - not necessarily in this PR

Reply:
Create https://github.com/iterative/dvc.org/issues/4787

Reply:
I would always use `:`. It's a bug to my mind that people don't use `:` before images and a paragraph that explains it.